### ISSUE TITLE
Hey,guy,do u have the Obj-c version?

### ISSUE THREAD

**Issue by KittenYang** [278204/GooeyMenu#1]

**Comment by palforsberg** [comment]
No, I'm sorry. That's not available at the moment but should be pretty straight forward to convert it.

**Comment by KittenYang** [comment]
Can u tell me the principle of the animation briefly?

**Comment by palforsberg** [comment]
Sure. The gooey part is just a circle drawn with a path where I animate the topmost point to make it jiggle. And then it's just three ball objects animating out of that with a 'bridge' drawn between the ball and the gooey part just like your KYCuteView.
